packages/benchmark/src/benchmark.js

```diff
@@ -0,0 +1,39 @@
+const performance = { now: () => Date.now() * 1_000_000 };
```

Comment:
This is not representative of [`performance.now`](https://nodejs.org/api/perf_hooks.html#performancenow), which returns _milliseconds_.
```suggestion
const performance = { now: () => Date.now() };
```

Reply:
Umm you are right. The name and function are conflicting. 
We just want to measure the time in nano-seconds. So, if anything's gonna change it's the name of the function.

Reply:
Changed the name.